Makefile

```diff
@@ -1,11 +1,5 @@
 # Makefile for CodeJail
 
-test: test_no_proxy test_proxy
-
-test_no_proxy:
-	@echo "Running all tests with no proxy process"
-	CODEJAIL_PROXY=0 nosetests
-
-test_proxy:
-	@echo "Running all tests with proxy process"
-	CODEJAIL_PROXY=1 nosetests
```

Comment:
We may want to keep these for now, in case we hit problems getting all the tests passing in the Docker container and need to resort to manual setup.  Interestingly, the target is called `test` here but `tests` in the README; we should probably fix that also.